instrumentation/opentelemetry-instrumentation-confluent-kafka/README.rst

```diff
@@ -6,7 +6,7 @@ OpenTelemetry confluent-kafka Instrumentation
 .. |pypi| image:: https://badge.fury.io/py/opentelemetry-instrumentation-confluent-kafka.svg
    :target: https://pypi.org/project/opentelemetry-instrumentation-confluent-kafka/
 
-This library allows tracing requests made by the confluent-kafka library.
+This library allows tracing requests made by the confluent-kafka library. This OpenTelemetry library is not written or supported by Confluent.
```

Comment:
Curious as to why this needs to be explicitly called out?

Reply:
None of the instrumentation here is supported by any of the original authors of the libraries. If we want to call out that, I think it's better to put something generic about the whole contrib repo instrumentations instead of each instrumentation.

Reply:
Due to the name of the library some users are under the impression that the library is supported by Confluent, so its best to have something that provides that context to avoid the confusion